.github/workflows/ci-precommit.yml

```diff
@@ -32,3 +30,17 @@ jobs:
       - uses: apache/skywalking-eyes/header@main
         with:
           config: tools/ci/licenserc.yml
+
+  # TODO: If it causes too much time to every PR, split to another
+  # workflow file and filter by paths:
+  # - '.github/actions/sync-content/action.yml'
+  # - '.github/workflows/ci-precommit.yml'
+  # - 'tools/pytools/**'
+  sync-without-push:
+    name: Check synchronize content
+    runs-on: ubuntu-latest
+    timeout-minutes: 180
+    steps:
+      - uses: actions/checkout@v3
+      - name: Sync content without push
+        uses: ./.github/actions/sync-content
```

Comment:
how does it decide not to attempt to push?

Reply:
It's decided at:

https://github.com/apache/pulsar-site/blob/22f11e47f8f98d6816067e0730741b8e191d077b/tools/pytools/bin/site-updater.py#L37-L50

We pass the default value `auto` in the CI env and it pushes only if it's in the CI env && in the upstream repo && not triggered by a `pull_request` event.

But since we split workflows now, it's an alternative we pass explicit push=y or n as an argument.